jena/src/test/java/org/apache/commons/rdf/jena/DatasetJenaTest.java

```diff
@@ -7,7 +7,7 @@
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
  *
- *     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Accidental reformat of the license.

Reply:
Please, remove this changeset from the PR.